.travis.yml

```diff
@@ -8,8 +8,8 @@ before_script:
   - docker --version
   - docker-compose --version
   - docker-compose up -d
-
   - sudo pip install --upgrade requests
+  - sudo pip install --upgrade pip
```

Comment:
We definitely should upgrade pip before using it to install other packages. 😆 Swap those lines.

Reply:
Done.